spd/experiments/lm/ss_llama_config.yaml

```diff
@@ -26,16 +26,32 @@ identity_module_patterns: null
 sampling: "continuous"
 use_delta_component: true
 loss_metric_configs:
+
   - classname: "ImportanceMinimalityLoss"
     coeff: 0.0003
     pnorm: 2.0
     p_anneal_start_frac: 0.0
     p_anneal_final_p: 0.7
     p_anneal_end_frac: 1.0
+
   - classname: "StochasticReconLayerwiseLoss"
     coeff: 2.0
+
   - classname: "StochasticReconLoss"
     coeff: 0.2
+
+  - classname: "PGDReconSubsetLoss"
+    init: "random"
+    step_size: 0.01
+    n_steps: 3
+    mask_scope: "shared_across_batch"
+    coeff:
+      type: "cosine"
+      start_value: 0.0
+      end_value: 0.0
+      start_frac: 0.0
+      end_frac: 1.0
```

Comment:
Best remove from the config (the coeffs are 0 anyway). Can add it to the eval_loss_metrics instead